README.md

```diff
@@ -6,12 +6,16 @@ ATLAS Automation API allows an external application to control an instance of AT
 
 This solution contains the following samples: 
 1. C# application. 
-1. MATLAB script.
+2. MATLAB script.
 
-Both do the same - create a transient parameter, add a Waveform Display, then add vCar and the new parameter to it. 
+Both sample codes do the same<sup>1</sup>: creating a transient parameter, adding a Waveform Display, then `vCar` and the new parameter to it. 
 
-**Instructions**: To run start ATLAS, load a session with parameter vCar in it and execute the sample application. 
+**Instructions**: Before running, start ATLAS, load a session<sup>1</sup> with parameter `vCar` in it and execute the sample application<sup>2</sup>. 
```

Comment:
The sample will now always load a session, is it still necessary for a session to be loaded as a prerequisite?

Reply:
Still necessary for the Matlab code sample, as I've not made the change in there